docs/vhost-user-blk-testing.md

```diff
@@ -35,38 +35,58 @@ Here are some good references for detailing them.
 
 ## Test environment
 
-The below test environment is based on ubuntu release(16.04.1 LTS), as for other system, please check related document.
-The test runs with multiple queue (MQ) support enabled, using 4 queues defined for both SPDK and the virtual machine.
-Here are the details on how the test can be run.
-
-### The hugepages settings in host linux
-Add "default_hugepagesz=1G hugepagesz=1G hugepages=2" into host linux cmdline.
-As for how to change Ubuntu linux cmdline in grub file, please ref below link:
-https://www.ostechnix.com/configure-grub-2-boot-loader-settings-ubuntu-16-04/
-reboot Ubuntu
+The below test environment is based on Ubuntu release (20.04.1 LTS), as for other system, please check related documents.
+This test runs with multiple queue (MQ) support enabled, using 4 queues defined for both SPDK and the virtual machine.
+Here are the detailed instructions.
+
+### The hugepages settings on the host
+Add the following options to the default kernel boot parameter on the
+host (to the grub config file `/etc/default/grub` as instructed
+[here](https://wiki.ubuntu.com/Kernel/KernelBootParameters)):
+```
+# Pre-allocate hugepages on system boot
+GRUB_CMDLINE_LINUX_DEFAULT="default_hugepagesz=1G hugepagesz=1G hugepages=2"
+```
+
+Update grub, reboot the system, and create the mount for hugepages (if it is not available):
```

Comment:
That sounds like a lot to modify grub in order to force the kernel to boot with the right amount of hugepages. There's an easier way to allocate hugepages on a host already booted:
```
# echo 512 > /proc/sys/vm/nr_hugepages
```

Reply:
Using host kernel command line is the only way to allocate 1GiB huge pages on the host.

Reply:
Oh because you mean you cannot change the page size at runtime?

Reply:
The kernel maintains multiple pools of huge pages of different page sizes. The only way to change the size for the default pool is with `default_hugepagesz=1G` or recompiling. The only way to reserve pools in non-default huge page size pool is with pairs of `hugepagesz=<sz> hugepages=<n>`. So in this particular case the kernel command line could be simplified to either just update the default huge page size and allocate at runtime or just fill the 1GiB pool with 2 pages.

Reply:
@likebreath The rest of our instructions about hugepages use 2MiB huge pages so for consistency it might be better to drop what is here with 1GiB huge pages and just go with 2MiB huge pages.